extensions/logging-gelf/runtime/src/main/java/io/quarkus/logging/gelf/GelfConfig.java

```diff
@@ -61,4 +64,32 @@
      */
     @ConfigItem(defaultValue = "yyyy-MM-dd HH:mm:ss,SSS")
     public String timestampPattern;
+
+    /**
+     * The logging-gelf log level.
+     */
+    @ConfigItem(defaultValue = "ALL")
+    Level level;
+
+    /**
+     * Name of the Facility.
+     */
+    @ConfigItem(defaultValue = "jboss-logmanager")
```

Comment:
Is it possible to have an enum here?

Reply:
I don't think an enum will be OK here.
The facility shoud be the name of the logging system, that's why the default value is `jboss-logmanager` but user can override it to whatever they want (typical use case: the name of the application)

Reply:
Oh okay, thanks for the explanation.